

package ContactInfo;

message Person {
  required string curName = 1;
  required int32 curId = 2;
  optional string curEmail = 3;

  enum PhoneType {
    MOBILE = 0;
    HOME = 1;
    WORK = 2;
  }

  message PhoneNumber {
    required string number = 4;
    optional PhoneType type = 2 [default = HOME];
  }

  repeated PhoneNumber phone = 4;
}

message AddressBook {
    required string owner = 10;
    repeated Person personInfo = 6;
}



